examples/Q-TCP/q_tcp.py

```diff
@@ -288,25 +291,21 @@ def qubit_recv_w_retransmission(host, q_size, sender_id, checksum_size_per_qubit
         if checksum_qubits[i].measure() != 0:
             errors += 1
 
-    # latest_seq_num = host.get_sequence_number(sender_id)
-    # print('LATEST SEQ NUM 4')
-    # print(latest_seq_num)
-
-    print('---------')
+    Logger.get_instance().log('---------')
```

Comment:
Probably better to keep these as a print statement since they're part of an example and not system messages. Also applying to the other Logger uses in the example

Reply:
@mehmetmertbese 
Have you had a chance to change these to print statements?